servers/plain/src/main/docker/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+# launch wildfly
+echo "launching wildfly"
+exec /opt/jboss/wildfly/bin/standalone.sh -b 0.0.0.0 $@
```

Comment:
If we're going to keep this file (which I think is not adding much value at this point), the path to the script should have `$JBOSS_HOME`, so that we don't have to have a different script on EAP (where the script would be in `$JBOSS_HOME/bin/standalone.sh`, which in that case is `/opt/eap/bin/standalone.sh`.

```suggestion
exec $JBOSS_HOME/bin/standalone.sh -b 0.0.0.0 $@
```